notifications.bs

```diff
@@ -102,15 +102,24 @@ or vibration pattern that is not otherwise accessible to the end user.
 
 <p>A <a lt="concept notification">notification</a> has an associated list of
 zero or more <dfn>actions</dfn>. Each action has an associated
-<dfn lt="action title">title</dfn> and <dfn>action name</dfn>. Users may
-activate actions, as alternatives to activating the notification itself. The
-user agent must determine the <dfn>maximum number of actions</dfn> supported,
-within the constraints of the notification platform.
+<dfn lt="action title">title</dfn> and <dfn>action name</dfn> and <em>can</em>
+have an associated <dfn>action icon URL</dfn> and
+<dfn>action icon resource</dfn>. Users may activate actions, as alternatives to
+activating the notification itself. The user agent must determine the
+<dfn>maximum number of actions</dfn> supported, within the constraints of the
+notification platform.
 
 <p class=note>Since display of actions is platform-dependent, developers are
 encouraged to make sure that any action a user can invoke from a notification is
 also available within the web application.
 
+<p class=note>On some platforms an <a>action icon resource</a> may be modified
+before being displayed to fit better with the platform's visual style. For
+example, it may be painted a specific color, corners may be rounded, a drop
+shadow may be applied. It is recommended to use an icon design that handles such
+modifications gracefully and does not lose important information through e.g.
+loss of color or clipped corners.
```

Comment:
"may be modified" is a bit ambiguous. What about the following phrasing?

> Some platforms may modify an [action item resource] to better match the platform's visual style
before displaying it to the user, for example by adding rounded corners or painting it in a specific
color. It is recommended to use an icon that handles such cases gracefully and does not lose
important information through e.g. loss of color or clipped corners.

It would be great if we could add something among the lines of "TEST!", but it's already quite verbose as-is.